tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -184,3 +186,15 @@ def test_rh_here_objects(self, cluster):
         save_test_table_remote()
         assert "test_table" in cluster.keys()
         assert isinstance(cluster.get("test_table"), rh.Table)
+
+    @pytest.mark.level("local")
+    def test_rh_status(self, cluster):
```

Comment:
This tests the local use case, but prob also worth adding a test where we run the command on the cluster: 
For example: 
`cluster.run(["runhouse status"])`